### ISSUE TITLE
[FEATURE]: Unit kWh not valid for deviceclass power

### ISSUE THREAD

**Issue by mvdwetering** [plugwise/plugwise-beta#369]
### Describe the solution you are looking for

I can't find a template to report a bug so using feature instead. If I need to report it elsewhere please let me know, but since I am using the Stick I suppose it should go here.

### Is your feature request related to a problem? (optional)

I just installed the `plugwise-beta` to use my Plugwise Circle(+) with Home Assistant through the stick. It seems to work as expected, but after enabling all sensors I noticed these errors in the Home Assistant logs menu.

```
Logger: homeassistant.components.sensor
Source: components/sensor/__init__.py:669
Integration: Sensor (documentation, issues)
First occurred: 16:47:37 (4 occurrences)
Last logged: 16:47:38

Entity sensor.power_consumption_current_hour_c2434 (<class 'custom_components.plugwise.sensor.USBSensor'>) is using native unit of measurement 'kWh' which is not a valid unit for the device class ('power') it is using; expected one of ['kW', 'W']; Please update your configuration if your entity is manually configured, otherwise report it to the custom integration author.
Entity sensor.power_production_current_hour_c2434 (<class 'custom_components.plugwise.sensor.USBSensor'>) is using native unit of measurement 'kWh' which is not a valid unit for the device class ('power') it is using; expected one of ['kW', 'W']; Please update your configuration if your entity is manually configured, otherwise report it to the custom integration author.
Entity sensor.power_consumption_current_hour_c461b (<class 'custom_components.plugwise.sensor.USBSensor'>) is using native unit of measurement 'kWh' which is not a valid unit for the device class ('power') it is using; expected one of ['kW', 'W']; Please update your configuration if your entity is manually configured, otherwise report it to the custom integration author.
Entity sensor.power_production_current_hour_c461b (<class 'custom_components.plugwise.sensor.USBSensor'>) is using native unit of measurement 'kWh' which is not a valid unit for the device class ('power') it is using; expected one of ['kW', 'W']; Please update your configuration if your entity is manually configured, otherwise report it to the custom integration author.
``` 

It seems to indicate that either the unit or deviceclass for some sensors is wrong.

### What type of installation are you running?

Home Assistant OS

### How did you install plugwise-beta?

HACS

### What kind of Plugwise device is having issues?

USB: Stick

### What firmware version is your Plugwise product at?

Circle is at 2011-06-27 10:52:18
Circle+ is at 2011-06-27 10:47:37
Stick I don't know as it is not shown as a device in Home Assistant

### Your setup

USB stick, Circle+ and Circle

**Comment by bouwew** [comment]
Thanks, this will be fixed in plugwise-beta v0.34.8, to be released during the coming days.

**Comment by bouwew** [comment]
If you want to test, please test the prerelease v0.34.8b2, this version contains the fixes.

**Comment by bouwew** [comment]
Ah yes, we have seen this before.
I will remove the device_class so that they can keep the state_class = measurements which means the data can be stored with history.

**Comment by bouwew** [comment]
Before I make any more changes, how do the history-graphs of these sensors look like?
One fixed value per hour/day? 
Or increasing values per hour/day like this? 
![image](https://user-images.githubusercontent.com/11290930/228600227-8eda7072-4850-4035-89e6-8966624192db.png)

**Comment by mvdwetering** [comment]
This is how it looks for me on the history page. 
![plugwise_power](https://user-images.githubusercontent.com/732514/229104833-68b3b7f8-64aa-4cc7-8439-2e347d7fde42.png)

Strangely enough only the "Power consumption current hour" seems to do something (production is expected to do nothing as I don't produce on the plugs). The other consumptions (previous hour/today/yesterday) seem to be stuck at 0. The "day" ones might be due to me reinstalling the integration and it has not been running for a day yet.

**Comment by bouwew** [comment]
Yes, I have enabled some of the sensors in my own system. I see the same for the `_current_,,,` and `_today_` sensors.
So these will become sensors with state_class measurements.
But I see that the `_previous_...` sensors are different: the provide one update per time-interval (hour/day). These should become sensors with state_class = Total.

**Comment by bouwew** [comment]
Fixed via #376
